assets/scss/_variables.scss

```diff
@@ -52,9 +52,9 @@ $td-sidebar-border-color: $border-color !default;
 // These colors are all part of the theme palette, but the mix is fairly random to create variation. This can be overridden by the project if needed.
 $td-box-colors: $dark, $primary, $secondary, $info, $gray-600, $success, $warning, $dark, $danger, $primary, $secondary, $info !default;
 
-$link-color: darken($blue, 15%) !default;
+$link-color: adjust-color($blue, $lightness: -15%) !default;
```

Comment:
While I followed BS's upgrade path and generally replaced `darken()` by `shade-color()`, in this case the resulting color didn't look as good, so I opted for a call to `adjust-color()` --matching the previous color exactly.